src/main/jbake/content/comparison.adoc

```diff
@@ -16,15 +16,15 @@ The mapping between the specifications and the respective Apache TomEE versions
 [options="header",cols="5*1,2"]
 |===
 |TomEE version|Based on Tomcat|Java{nbsp}SE version|Micro{nbsp}Profile version|Jakarta{nbsp}EE version|Full list of specifications versions and implementations choices
-//|10.x  |10.1.x|11|TBA|10.0|xref:tomee-10.0/docs/comparison.adoc#specifications[specifications versions] tomee-10.0/docs/comparison.adoc#implementations[implementations details]
-| 9.x  |10.0.x|11|5.0| 9.1|xref:tomee-9.0/docs/comparison.adoc#specifications[specifications versions] xref:tomee-9.0/docs/comparison.adoc#implementations[implementations details]
+|10.x  |11.0.x|17|6.0|10.0|//xref:tomee-10.0/docs/comparison.adoc#specifications[specifications versions] xref:tomee-10.0/docs/comparison.adoc#implementations[implementations details]
+| 9.x  |10.1.x|11|5.0| 9.1|xref:tomee-9.0/docs/comparison.adoc#specifications[specifications versions] xref:tomee-9.0/docs/comparison.adoc#implementations[implementations details]
```

Comment:
TomEE 9.1.x is based on Tomcat 10.0.x (or exactly 10.0.27 because Tomcat 10.0.x is EOL)